connector_flow/task/ftp_download.py

```diff
@@ -0,0 +1,111 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2014 initOS GmbH & Co. KG (<http://www.initos.com>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import models, api
+
+from .abstract_task import AbstractTask
+import ftputil
```

Comment:
@codingforfun Since the dependency is declared in the manifest,  Odoo will prevent the installation if it's not present in the pythonpath. But don't forget that even if your module is not installed, it will be imported into the runtime and thus the server won't start if the ftputil lib is not installed. That's why you should protect the import in a try catch to allow the server to start without the lib installed if your module is not installed.